.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce the behavior
+
+### Environment
+- rsyslog version: [e.g. 8.31.0 -- see rsyslogd -v]
+- platform: [e.g. Ubuntu 16.04]
+- for configuration questions/issues, include rsyslog.conf:
```

Comment:
I would be tempted to have the configuration be its own section, just to stress how important it is.

That said, I can understand that a compromise has to be made between providing a "survey" version of the template and one that follows the "keep it brief and to the point" philosophy.